lib/commands/users.js

```diff
@@ -1820,42 +2009,80 @@ users
 
 users
     .command(`update-mfa`)
-    .description(`Enable or disable MFA on a user account.`)
+    .description(`[**DEPRECATED** - This command is deprecated. Please use 'users update-mfa' instead] Enable or disable MFA on a user account.`)
```

Comment:
this us command `users` `update-mfa`. Says its deprecated and to use `users update-mfa`. I think this is not right

Reply:
I then see another definition just below, for same command